cmd/peer-id-to-pub-key/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"encoding/hex"
+	"fmt"
+	"log"
+	"os"
+
+	peer "github.com/libp2p/go-libp2p-peer"
```

Comment:
"github.com/libp2p/go-libp2p-peer" is deprecated. Use this instead:

```suggestion
	peer "github.com/libp2p/go-libp2p-core/peer"
```